packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -372,10 +388,10 @@ export class RunningSummarizer implements IDisposable {
 	/** We only want a single heuristic runner micro-task (will provide better optimized grouping of ops) */
 	private heuristicRunnerMicroTaskExists = false;
 
-	public handleOp(op: ISequencedDocumentMessage) {
+	public handleOp(op: ISequencedDocumentMessage, runtimeMessage?: boolean) {
 		this.heuristicData.lastOpSequenceNumber = op.sequenceNumber;
 
-		if (isRuntimeMessage(op)) {
+		if (runtimeMessage || isRuntimeMessage(op)) {
```

Comment:
Curious, why do we need both checks? Feels like they should be equivalent (looking at implementation of isRuntimeMessage()).

Reply:
Same below

Reply:
They are not equivalent in this case. The `runtimeMessage` boolean from the `ContainerRuntime` `"op"` event needs to be used to determine this. Because the op is unpacked and won't have the type "op", but will be something else (i.e. `ContainerMessageType`)

That said, I'll clean it up so the deltaManager event handler will do this `isRuntimeMessage` check and pass in a boolean. Intention should be clearer like that.